.eslintrc.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
   rules: {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
```

Comment:
Finally, we don't need to disabled this rule, isn't it?

Reply:
you're right